src/bundles/ApplicationsAdmin/components/AppList/AppList.js

```diff
@@ -73,7 +73,8 @@ class AppList extends Component {
             </div>
             <div className="col-sm-4 col-xs-12">
               <label htmlFor="keyword">Search:</label>
-              <input id="keyword" type="text" size="30" placeholder="id or name" onChange={onKeywordChange}/>
+              <input id="keyword" type="text" size="30" onChange={onKeywordChange}/>
+              <p style={{'font-size':'10px'}}>Search by: Profile Name, Application ID, Email Domain</p>
```

Comment:
```suggestion
              <p style={{'font-size':'13px'}}>Search by: Profile Name, Application ID, Email Domain</p>
```
```suggestion
              <p style={{'font-size':'10px'}}>Search by: Profile Name, Application ID, Email Domain</p>
```
10px is a bit too small. Lets change it to 13px